modules/tinymce/src/plugins/accordion/test/ts/webdriver/AccordionBackspaceDeleteTest.ts

```diff
@@ -29,6 +30,48 @@ describe('webdriver.tinymce.plugins.accordion.AccordionBackspaceDeleteTest', ()
   const pDoBackspace = () => pDoBackspaceDelete('Backspace');
   const pDoDelete = () => pDoBackspaceDelete('Delete');
 
+  context('Backspace should not remove accordion elements', () => {
+    it('TINY-9731: Prevent BACKSPACE from removing accordion body if a cursor is after the accordion', async () => {
+      const editor = hook.editor();
+      editor.setContent(AccordionUtils.createAccordion({ body: '<p><br/></p>' }) + '<p><br/></p>');
+      TinySelections.setCursor(editor, [ 1, 0 ], 0);
+      await pDoBackspace();
+      TinyAssertions.assertContentPresence(editor, { 'details > p': 1 });
+      TinyAssertions.assertCursor(editor, [ 0, 1 ], 0);
+    });
+
+    it('TINY-9884: Prevent BACKSPACE from removing accordion body if a cursor is in the accordion body', async () => {
+      const editor = hook.editor();
+      editor.setContent(AccordionUtils.createAccordion({ body: '<p><br/></p>' }));
+      TinySelections.setCursor(editor, [ 0, 1 ], 0);
+      await pDoBackspace();
+      TinyAssertions.assertContentPresence(editor, { 'details > p': 1 });
+      TinyAssertions.assertCursor(editor, [ 0, 1 ], 0);
+    });
+
+    it('TINY-9884: Prevent BACKSPACE from removing summary', async () => {
+      if (PlatformDetection.detect().browser.isFirefox()) {
+        // TODO - TINY-9949: Firefox performs an incorrect selection which causes the summary to be
+        // removed unexpectedly, even though it should not be possible.
+        return;
+      }
```

Comment:
To be consistent with how we usually do it
```suggestion
    it('TINY-9884: Prevent BACKSPACE from removing summary', async function () {
      if (PlatformDetection.detect().browser.isFirefox()) {
        // TODO - TINY-9949: Firefox performs an incorrect selection which causes the summary to be
        // removed unexpectedly, even though it should not be possible.
        this.skip();
      }
```